lua/orgmode/files/file.lua

```diff
@@ -269,6 +269,21 @@ function OrgFile:find_headline_by_title(title)
   end)
 end
 
+function OrgFile:get_todo_keywords()
```

Comment:
Let's memoize this so it's not constantly recalculated:

```suggestion
memoize('get_todo_keywords')
function OrgFile:get_todo_keywords()
```